eng/scripts/smoketest/Get-Package-Version.ps1

```diff
@@ -0,0 +1,83 @@
+param(
+    [string]$ArtifactsPath
+)
+. (Join-Path $PSScriptRoot ../../common/scripts/common.ps1)
+$ErrorActionPreference = 'Continue'
+
+$NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v2"
```

Comment:
Shouldn't this be:  
https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json

That's what we advertise in the [artifacts feed](https://dev.azure.com/azure-sdk/public/_packaging?_a=connect&feed=azure-sdk-for-net).

Reply:
For some reason the PS package cmdlets don't work with the v3 nuget endpoint they require the v2 endpoint.

Reply:
> For some reason the PS package cmdlets don't work with the v3 nuget endpoint they require the v2 endpoint.

The `PackageManagement` module in PowerShell is deprecated and hasn't been in active development for a while.  The PowerShell team recommends that it not be used.   There's some additional context in #26695.